### PR TITLE
LPAL-1152 Happy path govpay payment testing

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-1152](https://opgtransform.atlassian.net/browse/LPAL-1152)

## Approach

See ticket.

NB branch is the one I used for the spike.

## Learning

n/a

## Checklist

* [x] I have performed a self-review of my own code
* [x] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [x] I have added mandatory tags to terraformed resources, where possible
* [x] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [x] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

[LPAL-1152]: https://opgtransform.atlassian.net/browse/LPAL-1152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cypress/e2e/Errors.feature [hunk 1]
```diff
@@ -19,6 +19,7 @@ Feature: Errors
     # the same path through the code as invalid form fields (see below);
     # a failed login (but valid email input) produces a different type of error
     # from invalid form fields
+    @ErrorSummaryCheckAfter
```
Comment (townxelliot) [inline cypress/e2e/Errors.feature:22]: @RunErrorSummaryCheckAfterStep

File: cypress/e2e/LpaTypePF.feature [hunk 2]
```diff
@@ -22,7 +22,7 @@ Feature: Property and Finance LPA starting from the Type page
         When I click "add-donor"
         Then I can find "form-donor"
 
-    @focus @CleanupFixtures
+    @CleanupFixtures @EncodingCheckAfter @LinkCheckAfter
```
Comment (townxelliot) [inline cypress/e2e/LpaTypePF.feature:25]: @RunEncodingCheckAfterStep @RunLinkCheckAfterStep

File: cypress/e2e/common/after_checks.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const { After } = require('@badeball/cypress-cucumber-preprocessor');
+
+// NOTE: fixtures.js contains code for creating and cleaning fixtures,
+// which uses Before()/After() steps
+
+// After loading a page, check that any links with target="_blank" also
+// have rel="noreferrer noopener"; this is to prevent reverse tabnapping and
+// is recommended by the GOV.UK design system guidelines
+// https://design-system.service.gov.uk/styles/typography/#links
+After({ tags: '@LinkCheckAfter' }, () => {
```
Comment (townxelliot) [inline cypress/e2e/common/after_checks.js:10]: @RunLinkCheckAfterStep

**Comment by townxelliot** [comment]
All issues raised by me have now been addressed.

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**